packages/web-components/src/menu-item/menu-item.ts

```diff
@@ -55,6 +81,15 @@ export class MenuItem extends FASTElement {
   @attr
   public role: MenuItemRole = MenuItemRole.menuitem;
 
+  /**
+   * Handles changes to role attribute element internals properties
+   * @param prev - the previous state
+   * @param next - the next state
+   */
+  public roleChanged(prev: MenuItemRole | undefined, next: MenuItemRole | undefined) {
+    this.elementInternals.role = next ?? MenuItemRole.menuitem;
```

Comment:
Would this ever not be `MenuItemRole.menuitem`?

Reply:
Not by default - if next is null or undefined we want it to reset to `menuitem`